homeassistant/components/whirlpool/__init__.py

```diff
@@ -69,3 +71,4 @@ class WhirlpoolData:
     appliances_manager: AppliancesManager
     auth: Auth
     backend_selector: BackendSelector
+    session: ClientSession
```

Comment:
instead of storing the session here, couldn't platforms get it using async_get_clientsession() instead?

Reply:
We only need one session for our integration - If I understand the code correctly, you get a new session for each call.